WeakAuras/GenericTrigger.lua

```diff
@@ -3365,6 +3365,7 @@ function WeakAuras.WatchUnitChange(unit)
   if not watchUnitChange then
     ---@class UnitChangeFrame: FrameScriptObject
     watchUnitChange = CreateFrame("Frame");
+    watchUnitChange.trackedUnits = {}
```

Comment:
I've added this to quickly check whether we are even interested in this specific unit. This avoids e.g. tracking all nameplates if there's no unit characteristics trigger for nameplates.